src/ethereum/ethNetwork.js

```diff
@@ -751,14 +795,21 @@ export class EthereumNetwork {
             return { server, result }
           }
         )
-        funcs2 = async () => {
-          const result = await this.fetchPostInfura('eth_getTransactionCount', [
-            params[0],
-            'latest'
-          ])
-          return { server: 'infura', result }
+        if (infuraServers.length > 0) {
+          funcs2 = async () => {
+            const adjustedParams =
+              this.currencyInfo.currencyCode === 'RBTC'
```

Comment:
Wondering what you guys think of the reference to 'RBTC'. For some reason their Infura nodes expect the `param` array to be nested within an array (at least that's what works with the node that we are using).

Reply:
Yuck. Put a flag in the currenyInfo